nursery/encrypt-data-using-fakem-cipher.yml

```diff
@@ -0,0 +1,18 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: encrypt data using FAKEM cipher
```

Comment:
is FAKEM a family or encryption algorithm?

Reply:
`FAKEM` is a malware family - I'll add a description to make this more clear.